scripts/ts-function-release.sh

```diff
@@ -55,10 +55,7 @@ case "$2" in
         cd "${scripts_dir}/../functions/ts/${CURRENT_FUNCTION}"
 esac
 
-# This make it work for npm 6.*.*
 # https://github.com/GoogleContainerTools/kpt/issues/1394
```

Comment:
Do we still need this comment?

Reply:
> https://github.com/GoogleContainerTools/kpt/issues/1394

Yes, we still need this comment. The underlying issue is not resolved.